src/tests/unit/platform/test_asset_groups.py

```diff
@@ -321,3 +322,36 @@ def on_post(url, body, **kwargs):
     device = api.select(Device, 98765)
     group.remove_members(device)
     group.remove_members([device])
+
+
+def test_get_all_groups(cbcsdk_mock):
+    """Tests the get_all_groups class method."""
+    cbcsdk_mock.mock_request('GET', '/asset_groups/v1/orgs/test/groups', copy.deepcopy(GET_ALL_RESPONSE))
+    api = cbcsdk_mock.api
+    rc = AssetGroup.get_all_groups(api)
+    assert len(rc) == 2
+    assert isinstance(rc[0], AssetGroup)
+    assert isinstance(rc[1], AssetGroup)
+    assert rc[0].id == "db416fa2-d5f2-4fb5-8a5e-cd89f6ecda16"
+    assert rc[1].id == "509f437f-6b9a-4b8e-996e-9183b35f9069"
+
+
+def test_get_statistics(cbcsdk_mock):
+    """Tests the get_statistics method."""
+    cbcsdk_mock.mock_request('GET', '/asset_groups/v1/orgs/test/groups/db416fa2-d5f2-4fb5-8a5e-cd89f6ecda16',
+                             copy.deepcopy(EXISTING_AG_DATA))
+    cbcsdk_mock.mock_request('GET',
+                             '/asset_groups/v1/orgs/test/groups/db416fa2-d5f2-4fb5-8a5e-cd89f6ecda16/membership_summary',  # noqa: E501
+                             GET_STATS_RESPONSE)
+    api = cbcsdk_mock.api
+    group = api.select(AssetGroup, 'db416fa2-d5f2-4fb5-8a5e-cd89f6ecda16')
+    rc = group.get_statistics()
+    sub = rc["unassigned_properties"]
+    assert len(sub) == 1
+    assert sub[0]["type"] == "POLICY"
+    sub = rc["intersections"]
+    assert len(sub) == 2
+    assert sub[0]["count"] == 2
+    assert sub[0]["group_id"] == "509f437f-6b9a-4b8e-996e-9183b35f9069"
+    assert sub[1]["count"] == 1
+    assert sub[1]["group_id"] == "8e0e3714-fece-4c76-9728-6ad2713cde72"
```

Comment:
Instead of using a bunch of asserts why not assert the `rc == GET_STATS_RESPONSE` to make the test more stable if someone was to modify GET_STATS_RESPONSE

Reply:
Well, that makes me feel like we're not testing a whole hell of a lot here, but I can go with that.